ia/src/info.py

```
@@ -0,0 +1,23 @@
+import sys
+
+
+def info():
```

Ruff (0.11.9)

Missing return type annotation for public function `info`

Add return type annotation: `None`

(ANN201)